websites/J/JoJo's Bizarre Encyclopedia/presence.ts

```diff
@@ -16,61 +19,107 @@ presence.on("UpdateData", () => {
 		if (
 			window.location.pathname === "" ||
 			window.location.pathname === "/" ||
-			window.location.pathname === "/JoJo_Wiki"
+			window.location.pathname === "/JoJo_Wiki" ||
+			(window.location.pathname === "/index.php" &&
+				!url.searchParams.get("action"))
```

Comment:
`window.location` is used very often in this presence. It also looks like you created a URL object from the href.
For consistency/cleanliness, it may be a good idea to destructure the properties from either variable:
```ts
const { pathname, href, hostname /* etc */ } = window.location;
const { pathname, href, hostname /* etc */ } = url;
```

Reply:
You're right! I think I had planned to change ``window.location`` to ``url`` but forgot. I didn't think of destructuring it though. I've updated the file.